tests/fixtures/myst_role.md

```diff
@@ -3,14 +3,30 @@ Basic:
 .
 {abc}`xyz`
 .
-<p><code class="sphinx-role">{abc}[xyz]</code></p>
+<p><code class="myst role">{abc}[xyz]</code></p>
+.
+
+Multiple:
+.
+{abc}`xyz`{def}`uvw`
+.
+<p><code class="myst role">{abc}[xyz]</code><code class="myst role">{def}[uvw]</code></p>
 .
 
 Surrounding Text:
 .
 a {abc}`xyz` b
 .
-<p>a <code class="sphinx-role">{abc}[xyz]</code> b</p>
+<p>a <code class="myst role">{abc}[xyz]</code> b</p>
+.
+
+New lines:
```

Comment:
@webknjaz that test is already here. Is there any other fixtures you can think of before I merge

Reply:
How about a more complex case with spaces in the reference between angled braces?
```md
{ref}`some
text
<and
a
custom reference>`
```

Reply:
thanks for the quick reply

This complexity won't particularly make any difference here though: the markdown-plugin just stores everything inside the ticks as a raw string (now replacing new lines with spaces), then the token (with name+content) is processed specifically by myst-parser. i.e. it doesn't really matter what is inside the ticks (in the example above the token's content will be `some text <and a custom reference>`)

Reply:
Got it. But I'd still have some regression test for this, if not in this lib, then in some other place maybe?

Reply:
Yep, I can add a test like this in myst-parser, when I downstream this change to there 👍